internal/compiler/tests/syntax/analysis/binding_loop_layout4.slint

```diff
@@ -0,0 +1,39 @@
+// Copyright © SixtyFPS GmbH <info@slint-ui.com>
+// SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-commercial
+
+component Foo {
+//            ^error{The binding for the property 'layoutinfo-h' is part of a binding loop}
+//            ^^error{The binding for the property 'layoutinfo-h' is part of a binding loop}
+    HorizontalLayout {
+//  ^error{The binding for the property 'layout-cache' is part of a binding loop}
+//  ^^error{The binding for the property 'width' is part of a binding loop}
+//  ^^^error{The binding for the property 'width' is part of a binding loop}
+//  ^^^^error{The binding for the property 'layoutinfo-h' is part of a binding loop}
+//  ^^^^^error{The binding for the property 'layout-cache' is part of a binding loop}
+//  ^^^^^^error{The binding for the property 'width' is part of a binding loop}
+//  ^^^^^^^error{The binding for the property 'width' is part of a binding loop}
+//  ^^^^^^^^error{The binding for the property 'layoutinfo-h' is part of a binding loop}
```

Comment:
yeah, that's a lot of error on the same line!
Only gets worse when more components are involved in a binding loop.